not/localization/__init__.py

```diff
@@ -0,0 +1,98 @@
+"""The localization module provides localized user-facing strings
+
+generally, one need only import the polyglot singleton object
+
+This module uses an unorthodox approach to singletons to avoid keeping
+state in the module, which feels more awkward.
+
+When python 3.9 comes out in earnest, revisit using:
```

Comment:
should mention that we will probably use a back port because requiring 3.9 is a little too hard in the paint.